### PR TITLE
Features/check output by cols

### LINKED ISSUE #127: Antares Test : update python script to check column by column
In pytest `output_compare_test.py` the result matrix is compared.

This could be nice to check the values by column instead of full matrix. 
In this case we could : 
- parse header to get variable name to indicate variable name in case of error
- for some variable where there is a value mismatch between windows and linux, define a larger tolerance

### PR DESCRIPTION
**Pull request by jmkerloch**
closes #127

### CODE REVIEW

File: src/tests/examples/output_compare_test.py [hunk 1]
```diff
@@ -102,16 +107,43 @@ def compare_directory(result_dir, reference_dir):
                 compare_directory(x, reference_dir / x.name)
         else:
 
-            if x.name != 'id-daily.txt' :
+            uncompared_file_name = ['id-daily.txt', 'id-hourly.txt']
```
Comment (guilpier-code) [inline src/tests/examples/output_compare_test.py:110]: variable **dir_list**, at beginning of the function, is ununsed.

File: src/tests/examples/output_compare_test.py [hunk 2]
```diff
@@ -102,16 +107,43 @@ def compare_directory(result_dir, reference_dir):
...
-    result_dir = find_output_result_dir(path / 'output')
-    reference_dir = find_output_result_dir(path / 'reference')
+                
+                np.testing.assert_equal(reference_headers,output_headers, err_msg="headers dismatch in " + str(reference_dir / x.name), verbose=True)
+
+                rtol_override = {"CO2 EMIS." : 1e-3, "FLOW LIN." : 1e-3 , "UCAP LIN." : 1e-3, "H. INFL" : 1e-3 , "H. STOR" : 1e-3 , "H. OVFL" : 1e-3 , "OV. COST" : 1e-3 , "LIGNITE" : 1e-3 , "CONG. FEE (ABS.)" : 1e-3 , "sb" : 1e-3 , "MARG. COST" : 1e-3 , "DTG MRG" : 1e-3 , "BALANCE" : 1e-3 , "BASE" : 1e-3 , "MRG. PRICE" : 1e-3 , "OP. COST" : 1e-3 , "SEMI BASE" : 1e-3 ,"COAL" : 1e-3 , "MAX MRG" : 1e-3 , "UNSP. ENRG" : 1e-3}
+                atol_override = {"CO2 EMIS." : 1, "CONG. FEE (ALG.)" : 1, "FLOW LIN." : 1, "UCAP LIN." : 1, "peak" : 1, "PEAK" : 1, "H. INFL" : 1, "H. STOR" : 1, "HURDLE COST" : 1, "H. OVFL" : 1 , "LOAD" : 1, "CONG. FEE (ABS.)" : 1 , "sb" : 1 , "MISC. DTG" : 1 , "DTG MRG" : 1 , "BALANCE" : 1 , "BASE" : 1 , "OP. COST" : 1 , "SEMI BASE" : 1 , "COAL" : 1 , "p" : 1 , "MAX MRG" : 1 , "UNSP. ENRG" : 1 , "SOLAR" : 1 , "b" : 1 , "NODU" : 1 , "H. ROR" : 1}
```
Comment (guilpier-code) [inline src/tests/examples/output_compare_test.py:122]: Maybe **rtol_override** and **atol_override** as global variables. And renamed **rtol_override_for_Linux** and **atol_override_for_Linux**

**Review by guilpier-code — CHANGES_REQUESTED** [review]

File: src/tests/examples/output_compare_test.py (hunk 1 above)
Comment (jmkerloch) [reply src/tests/examples/output_compare_test.py:113]: done

File: src/tests/examples/output_compare_test.py (hunk 2 above)
Comment (jmkerloch) [reply src/tests/examples/output_compare_test.py]: done

**Review by guilpier-code — APPROVED** [review]

**Pull request merged**